draft-ietf-rats-endorsements.md

```diff
@@ -201,16 +201,40 @@ some minimal matching policy (e.g., exact match against a singleton reference
 value).  This unfortunately complicates design as a Verifier may need
 multiple parsers for matching policies.
 
-# Endorsing Identity
-
-One type of claims that might be endorsed would be claims having to do with
-identity, such as verification keys bound to an identifier for the Attesting Environment (e.g., {{-rats-eat}} `ueid`).  While identity claims are just another
+# Key Material for Attestation
+
+A Verifier receives Evidence from an Attester that must be considered untrusted
```

Comment:
```suggestion
A Verifier receives Evidence from an Attesting Environment of an Attester that must be considered untrusted
```

Reply:
This hurts readability and doesn't seem to add much in terms of precision.

Reply:
I agree with Thomas on this one, it's much less readable.